discovery/eureka_test.go

```diff
@@ -24,8 +24,10 @@ func TestMain(m *testing.M) {
 	resource, err := pool.RunWithOptions(&dockertest.RunOptions{
 		Repository:   "netflixoss/eureka",
 		Tag:          "1.3.1",
-		PortBindings: map[dc.Port][]dc.PortBinding{
-			dc.Port(strconv.Itoa(eurekaTestPort) + "/tcp"): {{HostIP: "", HostPort: strconv.Itoa(eurekaTestPort)}},
+		//Repository:         "containers.schibsted.io/spt-infrastructure/eureka-docker",
```

Comment:
what is this comment for? own testing?

Reply:
WIP. Comparing the (old) public version in eureka with one that is newer, but not public...